examples/cifar10/test.cpp

```diff
@@ -8,7 +8,7 @@ using namespace std;
 // rescale output to 0-100
 template <typename Activation>
 double rescale(double x) {
-  Activation a;
+  Activation a(1);
```

Comment:
Making an instance just to call `scale` method doesn't look right but the method can't be `static` since it's `virtual`.